modules/network-observability-metrics.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-metrics_{context}"]
+= Network Observability metrics
+Metrics generated by the `flowlogs-pipeline` are configurable in the `spec.processor.metrics.includeList` of the `FlowCollector` custom resource to add or remove metrics. 
+
+You can also create alerts by using the `includeList` metrics in Prometheus rules, as shown in the example "Creating an alert for high ingress traffic".
+
+When looking for these metrics in Prometheus, such as in the Console through Observe -> Metrics, or when defining alerts, all the metrics names are prefixed with "netobserv_". For example, "netobserv_namespace_flows_total". Available metrics names are as follows.
+
+.includeList metrics names
+[options="header"]
```

Comment:
Just checking - does this do anything?

Reply:
I don't think this [options="header"] actually does anything. I was thinking I needed it but I don't know why I was thinking that because I can't find anything in asciidoc writing reference or contributing guide that says we need this kind of thing. I've removed it to check and make sure.